lib/ssl/test/ssl_api_SUITE.erl

```diff
@@ -2996,8 +2996,8 @@ options_sign_alg(_Config) ->  %% signature_algs[_cert]
     ok.
 
 options_supported_groups(_Config) ->
-    %% FIXME group() type doesn't cover the values below
-    ?OK(#{supported_groups := {supported_groups, [x25519,x448,secp256r1,secp384r1]}},
+    Default = ssl:groups(default),
+    ?OK(#{supported_groups := {supported_groups, Default}},
         [], client),
     ?OK(#{supported_groups := {supported_groups, [secp521r1, ffdhe2048]}},
         [{supported_groups, [secp521r1, ffdhe2048]}], client),
```

Comment:
Are missing tests in options_eccs:   The option is now ignored for 1.3 if I understood you patch correctly?



```
 options_eccs(_Config) ->
    Curves = tl(ssl:eccs()),
    ?OK(#{eccs := {elliptic_curves, [_|_]}}, [], client),
    ?OK(#{eccs := {elliptic_curves, [_|_]}}, [{eccs, Curves}], client),

    %% Errors
    ?ERR({eccs, not_a_list}, [{eccs, not_a_list}], client),
    ?ERR({eccs, none_valid}, [{eccs, []}], client),
    ?ERR({eccs, none_valid}, [{eccs, [foo]}], client),
    ok.

```

Reply:
It has always been ignored by the TLS-1.3 code. So we could let it be any value without any affect.  But for OTP-27 we should consider making it configuration error to set it if only TLS-1.3 is supported.